### PR TITLE
Style: run the optimise imports, and code cleanup tool

### PR DESCRIPTION
**Pull request by Autek**
just run the tool code cleanup and optimise imports

### CODE REVIEW

File: .github/workflows/android.yaml [hunk 1]
```diff
@@ -53,13 +53,13 @@ jobs:
...
         run: ./gradlew assembleRelease
 
-#      - name: Sign the APK
-#        run: |
-#          jarsigner -verbose -sigalg SHA256withRSA -digestalg SHA-256 -keystore signIn.jks -storepass $KEYSTORE_PASSWORD -keypass $KEY_PASSWORD app/build/outputs/apk/release/app-release.apk $KEY_ALIAS
-#
-#      - name: Zipalign the APK
-#        run: |
-#          $ANDROID_HOME/build-tools/31.0.0/zipalign -v 4 app/build/outputs/apk/release/app-release.apk app/build/outputs/apk/release/app-M1.apk
+      #      - name: Sign the APK
```
Comment (JitterKiller) [inline .github/workflows/android.yaml:56]: I think you can remove those comments, it's not needed anymore to build the APK

File: emulator-data/auth_export/accounts.json
Comment (JitterKiller) [inline emulator-data/auth_export/accounts.json:1]: This is really good, it helps visualize the json in a better way

File: end2end-data/auth_export/accounts.json
Comment (JitterKiller) [inline end2end-data/auth_export/accounts.json:1]: Same here, great job

**Review by JitterKiller — DISMISSED** [review]
LGTM 👍

File: .github/workflows/android.yaml (hunk 1 above)
Comment (Autek) [reply .github/workflows/android.yaml]: You are right I will do that :)

**Review by JitterKiller — APPROVED** [review]
LGTM, this PR is now ready to be merged!

**Pull request merged**